src/pages/leaderboard.tsx

```diff
@@ -118,3 +118,13 @@ function isCurrentSeason(seasonId: number, seasons: Season[]): boolean {
   ) ?? { id: -1 };
   return currentSeasonId == seasonId;
 }
+
+function shortName(name: string): string {
+  const nameSplit = name.split(" ");
+
+  if (nameSplit.length > 1) {
+    return `${nameSplit[0]} ${nameSplit[1].substring(0, 1)}.`;
```

Comment:
```suggestion
    return `${nameSplit[0]} ${nameSplit.at(-1).substring(0, 1)}.`;
```

Would it make more sense to take the first letter of the last name rather than the second name?

Reply:
It would, I will fix that.